test/e2e-wi-tests.sh

```diff
@@ -36,10 +36,11 @@ function export_variable() {
     readonly DATA_PLANE_SERVICE_ACCOUNT_EMAIL=PUBSUB_SERVICE_ACCOUNT_EMAIL
     readonly PUBSUB_SERVICE_ACCOUNT_KEY_TEMP="${PUBSUB_SERVICE_ACCOUNT_NON_PROW_KEY_TEMP}"
   else
-    readonly CONTROL_PLANE_SERVICE_ACCOUNT_EMAIL=${PROW_SERVICE_ACCOUNT_EMAIL}
+    # readonly CONTROL_PLANE_SERVICE_ACCOUNT_EMAIL=${PROW_SERVICE_ACCOUNT_EMAIL}
     # Get the PROW service account.
     readonly PROW_PROJECT_NAME=$(cut -d'.' -f1 <<< "$(cut -d'@' -f2 <<< "${PROW_SERVICE_ACCOUNT_EMAIL}")")
-    readonly DATA_PLANE_SERVICE_ACCOUNT_EMAIL="cloud-run-events-source@${PROW_PROJECT_NAME}.iam.gserviceaccount.com"
+    readonly CONTROL_PLANE_SERVICE_ACCOUNT_EMAIL="${CONTROL_PLANE_SERVICE_ACCOUNT}@${PROW_PROJECT_NAME}.iam.gserviceaccount.com"
+    readonly DATA_PLANE_SERVICE_ACCOUNT_EMAIL="events-sources-gsa@${PROW_PROJECT_NAME}.iam.gserviceaccount.com"
     readonly PUBSUB_SERVICE_ACCOUNT_EMAIL=${PROW_SERVICE_ACCOUNT_EMAIL}
```

Comment:
We are using PUBSUB_SERVICE_ACCOUNT_EMAIL to grant permission for the broker in: https://github.com/google/knative-gcp/blob/f0c6500bfbcc1488fde44c571f8442666ada9c4e/test/e2e-common.sh#L119-L126. It seems we also need to update these line.